web/src/components/pages/dashboard/dashboard.css

```diff
@@ -217,7 +218,24 @@
         background-color: var(--light-grey);
     }
 }
-.inviteButton {
+.languages.challenge {
+    display: none !important;
+}
```

Comment:
I'd prefer a conditional render in the `.tsx` file – that way we don't clutter the DOM with unused components. I'll make that change in a future commit, don't worry about it!